piksi_tools/ardupilot/mavlink_decode.py

```diff
@@ -118,7 +299,7 @@ def rewrite(records, outfile):
            "data": dispatch(sbp).to_json_dict(),
            "metadata": {}}
```

Comment:
Current JSON format looks like this:

{"data": {"v_accuracy": 0, "n_sats": 0, "sender": 51763, "msg_type": 522, "lon": 0.0, "tow": 0, "height": 0.0, "crc": 37084, "length": 34, "flags": 0, "h_accuracy": 0, "lat": 0     .0, "preamble": 85, "payload": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="}, "session-uid": "9f2c012a-8a1c-44ef-a1ef-c32aef1904ce", "time": "2017-06-08T01:38:47.77178}

Can you make sure this output looks exactly like the output form a json log from the console?